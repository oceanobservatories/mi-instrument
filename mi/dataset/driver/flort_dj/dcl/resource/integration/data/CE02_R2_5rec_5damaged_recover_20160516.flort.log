2016/05/16 00:00:13.136 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2016/05/16 21:02:26.846 05/16/16	20:56:25	700	388	695	130	460	74	550
2016/05/16 21:02:27.976 05/16/16	20:56:26	451	695	129	460	74	550
2016/05/16 21:02:29.107 05/16/16	20:56:27	700	396	126	460	76	550
2016/05/16 21:02:30.235 05/16/16	20:56:28	700	421	695	134	460	550
2016/05/16 21:02:31.365 05/16/16	20:56:29	700	436	695	136	74	549
2016/05/16 21:02:32.504 05/16/16	20:56:30	700	396	695	460	77	549
2016/05/16 21:02:33.624 05/16/16	20:56:31	700	425	695	128	460	74	549
2016/05/16 21:02:34.754 05/16/16	20:56:33	700	369	695	132	460	74	550
2016/05/16 21:02:35.884 05/16/16	20:56:34	700	446	695	131	460	73	549
2016/05/16 21:02:37.013 05/16/16	20:56:35	700	436	695	131	460	76	549
2016/05/16 21:03:00.954 [flort:DLOGP2]:Instrument Stopped [Power Off]
