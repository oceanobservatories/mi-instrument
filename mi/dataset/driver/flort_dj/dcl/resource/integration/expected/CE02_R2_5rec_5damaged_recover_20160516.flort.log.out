
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE02SHSM | RID27 | 02-FLORTD000 | 3672345600 | recovered | 3672421346.8459997177 |          2 |         3672420985 | 3672421346.8459997177 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             388 |              74 |            130 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3672345600 | recovered | 3672421353.6240000725 |          2 |         3672420991 | 3672421353.6240000725 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               549 |             425 |              74 |            128 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3672345600 | recovered | 3672421354.7539997101 |          2 |         3672420993 | 3672421354.7539997101 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             369 |              74 |            132 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3672345600 | recovered | 3672421355.8839998245 |          2 |         3672420994 | 3672421355.8839998245 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               549 |             446 |              73 |            131 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3672345600 | recovered | 3672421357.0130000114 |          2 |         3672420995 | 3672421357.0130000114 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               549 |             436 |              76 |            131 |        null

(5 rows)
